src/lib/components/Error/Error.js

```diff
@@ -9,14 +9,13 @@ export const isAPIError = (error) => {
   return get(error, 'response.data.message') !== undefined;
 };
 
-const shouldShowErrorPage = (error) => {
-  if (isAPIError(error)) {
-    // do not display the error in a full page, show the error only in the specific component (probably a notification).
-    return false;
-  }
-  // when the exception is handled, we don't want to display a full page error (probably a notification).
+export const shouldShowErrorPage = (error) => {
   const handledException = error?.response?.status === 400;
-  return handledException ? false : true;
+  if (!handledException || !isAPIError(error)) {
```

Comment:
Feel free to keep this like this, the only comment I have is that, in general, to improve readability, we try avoiding double negation. It is more natural to read "positive" conditions.
Here you could:
```javascript
if (handledException || isAPIError(error)) {
  // don't display full page error
  return false; 
} else {
  return true;
}
```